app/assets/stylesheets/views/_landing_page/featured.scss

```diff
@@ -2,17 +2,28 @@
 
 .featured {
   display: flex;
-  background: govuk-colour("dark-blue");
 
   @include govuk-media-query($until: desktop) {
     flex-direction: column;
   }
 }
 
 .featured__child--content {
+  width: 66.666%;
+
+  @include govuk-media-query($until: desktop) {
+    width: 100%;
+  }
+}
+
+.featured__child--content-with-image {
   flex: 1 1 33.333%;
-  padding: govuk-spacing(6);
+  padding: 0 govuk-spacing(6) govuk-spacing(6) 0;
```

Comment:
I think this causes excess padding when viewed on mobile.

<img width="1075" alt="Screenshot 2024-11-21 at 13 39 30" src="https://github.com/user-attachments/assets/21f3a4af-2b5c-4440-b1b4-056da7012ed3">

We might need to set it like `padding: 0 0 30px;` initially, and then use a (desktop?) media query to add the `padding-right`.